packages/devtools_app/lib/src/inspector/flutter/summary_tree_debug_layout.dart

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+import '../../inspector/inspector_text_styles.dart' as inspector_text_styles;
+import '../diagnostics_node.dart';
+import 'inspector_data_models.dart';
+
+class ConstraintsDescription extends AnimatedWidget {
+  const ConstraintsDescription({
+    this.diagnostic,
+    AnimationController animationController,
```

Comment:
use the name listenable for the named parameter for consistency with AnimatedWidget